partiql-ast/src/main/java/org/partiql/ast/expr/Scope.java

```diff
@@ -14,13 +14,8 @@
  */
 @EqualsAndHashCode(callSuper = false)
 public class Scope extends AstEnum {
```

Comment:
Where is Scope used in the AST? This might be a holdover from @T vs T which I think shouldn't be "scope" but more like some kind of qualifier boolean on the ExprVar/ExprId or whatever we are calling it.

Reply:
Yeah, `ExprVarRef` is the only place it's used. In a future PR (probably when I update the docs), I'll remove the `Scope` enum opt for a qualifier boolean.